rustfmt-core/rustfmt-lib/tests/source/issue-2916.rs

```diff
@@ -0,0 +1,2 @@
+a_macro!(name<Param1, Param2>, 
```

Comment:
apparently the trailing whitespace is now actually being trimmed, so creating a test file under `source` directory with the trailing whitepsace to keep the test case valid since the end formatting result in the file under `target` has the whitepsace removed